content/kubermatic/master/cheat_sheets/vsphere_cluster_id/_index.en.md

```diff
@@ -0,0 +1,207 @@
+      provider, and therefore you're **not** affected. **Make sure that you 
+      don't use an affected KKP version when migrating the cluster to the
+      external CCM/CSI**
+
+## Changing the cluster-id property
+
+This guide shows two possible approaches to change the cluster-id.
+
+The first approach is [recommended by VMware][vmware-kb], however, it requires
+pausing the affected KKP user clusters and stopping the CSI driver for about
+one hour. For that time, KKP will not reconcile your affected clusters and
+you'll not be able to work with vSphere volumes (attach/detach volumes, create
+new volumes, delete volumes...).
+
+The second approach assumes changing `cluster-id` without stopping the CSI
+driver. This approach is not documented by VMware, however, it worked in our
+environment. In this case, there's no significant downtime. Since this approach
+is not recommended by VMware, we **heavily advise** that you:
+   - follow the first approach
+   - if you decide to follow this approach, make sure to extensively test it in
+     a staging/testing environment before applying it in the production
+
+### Approach 1
+
+{{% notice warning %}}
+This approach assumes pausing your affected KKP user clusters and stopping the
+vSphere CSI driver. When the KKP user cluster is paused, KKP controllers will
+not reconcile the cluster. You'll not be able to use the KKP dashboard to
+manage your cluster. You'll be able to use kubectl to access your cluster, and
+your workload will run as usual.
+{{% /notice %}}
+
+{{% notice warning %}}
+Stopping the CSI driver assumes that you'll not be able to work with volumes
+while it's stopped. This means that you can't attach/detach volumes, or
+create/delete volumes. This also means that you'll not be able to schedule/run
+new pods using vSphere volumes. It's strongly recommended that you don't delete
+any pods using vSphere volumes while this procedure is ongoing, or otherwise
+you might not be able to run new pods until the cluster is not unpaused.
+{{% /notice %}}
+
+This approach assumes doing this procedure for all affected user clusters **at
+the same time**. You can optionally choose one cluster and leave it with the
+old value, but we recommend migrating all your affected clusters.
+
+This approach is based on the [official recommendation from VMware][vmware-kb].
+
+1. Get the kubeconfig files for all affected user clusters
+
+2. Pause all affected user clusters. You can run the following `kubectl patch`
+   command for all affected user cluster:
+
+    ```shell
+    clusterPatch='{"spec":{"features":{"vsphereCSIClusterID":true}}}'
+    kubectl patch cluster <cluster-name> --type=merge -p $clusterPatch
+    ```
+
+3. Scale down the vSphere CSI driver deployment in all affected user cluster:
+
+    ```shell
+    export KUBECONFIG=<user-cluster-kubeconfig>
+    kubectl scale deployment -n kube-system vsphere-csi-controller --replicas=0
```

Comment:
since it's expected to iterate over different user-cluster-kubeconfigs, probably `--kubeconfig` flag is preferable.